### PR TITLE
[mainline]: updating docs after release/v1.22.0

### PR DESCRIPTION
**Pull request by abukhoy**
This is created for updating docs in order to make it aligned with the source code.

### CODE REVIEW

File: docs/source/introduction.md [hunk 1]
```diff
@@ -20,12 +20,25 @@ For other models, there is comprehensive documentation to inspire upon the chang
...
 
-***Latest news*** : <br>
 
-- [coming soon] Support for more popular [models](models_coming_soon)<br>
+*Latest news* :fire: <br>
+- [06/2026] Added support for Gemma4 models, [google/gemma-4-E2B-it](https://huggingface.co/google/gemma-4-E2B), [google/gemma-4-26B-A4B-it](https://huggingface.co/google/gemma-4-26B-A4B-it)
+- [06/2026] Added support for Qwen3.5 MoE model [Qwen/Qwen3.6-35B-A3B](https://huggingface.co/Qwen/Qwen3.6-35B-A3B)
+- [06/2026] Added support for Qwen3.5 model[Qwen/Qwen3.5-0.8B](https://huggingface.co/Qwen/Qwen3.5-0.8B)
+- [06/2026] Added aupport for Qwen3 VL and Qwen3 MoE VL models, [Qwen/Qwen3-VL-30B-A3B-Instruct](https://huggingface.co/Qwen/Qwen3-VL-30B-A3B-Instruct), [Qwen/Qwen3-VL-32B-Instruct](https://huggingface.co/Qwen/Qwen3-VL-32B-Instruct)
+- [06/2026] Added support for Qwen3 VL Reranker model, [Qwen/Qwen3-VL-Reranker-8B](https://huggingface.co/Qwen/Qwen3-VL-Reranker-8B)
```
Comment (quic-hemagnih) [inline docs/source/introduction.md]: remove reranker

File: docs/source/release_docs.md [hunk 2]
```diff
@@ -1,3 +1,233 @@
...
+
+- **Qwen3-VL Embedding Model**
+  - Executable via [`QEffAutoModel`](#QEffAutoModel)
+  - Supports `Qwen/Qwen3-VL-Embedding-8B`
+  - CPU vs AI100 MAD validated
+
+- **Qwen3-VL Reranker Models**
+  - Executable via [`QEFFAutoModelForImageTextToText`](#QEFFAutoModelForImageTextToText)
+  - Supports `Qwen/Qwen3-VL-Reranker-2B` and `Qwen/Qwen3-VL-Reranker-8B`
+  - [Reranker Example Scripts](https://github.com/quic/efficient-transformers/tree/main/examples/reranker)
```
Comment (quic-hemagnih) [inline docs/source/release_docs.md]: remove

File: README.md [hunk 3]
```diff
@@ -117,7 +126,7 @@ pip install dist/qefficient-0.0.1.dev0-py3-none-any.whl
 
 ``` 
 
-For more details about using ``QEfficient`` via Cloud AI 100 Apps SDK, visit [Linux Installation Guide](https://quic.github.io/efficient-transformers/source/installation.html)
+For more details about using ``QEfficient`` via Cloud AI 100/200 Apps SDK, visit [Linux Installation Guide](https://quic.github.io/efficient-transformers/source/installation.html)
```
Comment (quic-hemagnih) [inline README.md]: Cloud AIxx (AI100, AI200 and so on)

**Review by quic-hemagnih — CHANGES_REQUESTED** [review]
I don't see update in the QEfficient docs for changing AI100 to AIxx

File: docs/source/introduction.md [hunk 4]
```diff
@@ -20,12 +20,25 @@ For other models, there is comprehensive documentation to inspire upon the chang
 7. Sample example applications and demo notebooks
 8. Unit test templates.
 
-***Latest news*** : <br>
 
-- [coming soon] Support for more popular [models](models_coming_soon)<br>
+*Latest news* :fire: <br>
+- [06/2026] Added support for Gemma4 models, [google/gemma-4-E2B-it](https://huggingface.co/google/gemma-4-E2B), [google/gemma-4-26B-A4B-it](https://huggingface.co/google/gemma-4-26B-A4B-it)
+- [06/2026] Added support for Qwen3.5 MoE model [Qwen/Qwen3.6-35B-A3B](https://huggingface.co/Qwen/Qwen3.6-35B-A3B)
+- [06/2026] Added support for Qwen3.5 model[Qwen/Qwen3.5-0.8B](https://huggingface.co/Qwen/Qwen3.5-0.8B)
```
Comment (quic-rishinr) [inline docs/source/introduction.md:28]: Do add one 3.6 model as well

File: docs/source/introduction.md [hunk 5]
```diff
@@ -20,12 +20,25 @@ For other models, there is comprehensive documentation to inspire upon the chang
...
+- [06/2026] Added aupport for Qwen3 VL and Qwen3 MoE VL models, [Qwen/Qwen3-VL-30B-A3B-Instruct](https://huggingface.co/Qwen/Qwen3-VL-30B-A3B-Instruct), [Qwen/Qwen3-VL-32B-Instruct](https://huggingface.co/Qwen/Qwen3-VL-32B-Instruct)
+- [06/2026] Added support for Qwen3 VL Reranker model, [Qwen/Qwen3-VL-Reranker-8B](https://huggingface.co/Qwen/Qwen3-VL-Reranker-8B)
+- [06/2026] Added support for Kimi-K2 model [moonshotai/Kimi-K2-Thinking](https://huggingface.co/moonshotai/Kimi-K2-Thinking)
```
Comment (quic-rishinr) [inline docs/source/introduction.md]: remove kimi-k2 from the list. All functionalities are not enabled

File: docs/source/introduction.md [hunk 6]
```diff
@@ -52,17 +60,20 @@ For other models, there is comprehensive documentation to inspire upon the chang
 - [04/2025] Support for [GGUF model execution](https://github.com/quic/efficient-transformers/pull/368) (without quantized weights) 
 - [04/2025] Enabled FP8 model support on [replicate_kv_heads script](https://github.com/quic/efficient-transformers/tree/main/scripts/replicate_kv_head)
 - [04/2025] Added support for [gradient checkpointing](https://github.com/quic/efficient-transformers/pull/338) in the finetuning script
+- [04/2025] Added support of model `ibm-granite/granite-vision-3.2-2b`[ibm-granite/granite-vision-3.2-2b](https://huggingface.co/ibm-granite/granite-vision-3.2-2b)
```
Comment (quic-rishinr) [inline docs/source/introduction.md]: why we are updating older summary?

File: docs/source/release_docs.md [hunk 7]
```diff
@@ -1,3 +1,233 @@
...
+
+## Newly Supported Models
+
+- **Gemma 4 (Text + Multimodal)**
+  - Executable via [`QEFFAutoModelForImageTextToText`](#QEFFAutoModelForImageTextToText) (multimodal)
+  - Covers both dense and MoE configurations
+  - Optimized chunked prefill and disaggregated (vision/language split) serving
+  - [Gemma 4 Example Scripts](https://github.com/quic/efficient-transformers/tree/main/examples/image_text_to_text/models/gemma_vision/gemma4)
+
+- **Kimi-K2 (MLA + MoE)**
```
Comment (quic-rishinr) [inline docs/source/release_docs.md]: Lets remove kimi k2 for now. its not fully functional on release 1.22 branch

File: docs/source/release_docs.md [hunk 8]
```diff
@@ -1,3 +1,233 @@
...
+  - Continuous batching and disaggregated serving support
+  - [Qwen3.5 Example Scripts](https://github.com/quic/efficient-transformers/tree/main/examples/image_text_to_text/models/qwen3_5)
+
+- **GLM4-MOE**
+  - Executable via [`QEFFAutoModelForCausalLM`](#QEFFAutoModelForCausalLM)
+  - Disaggregated prefill and decode support with chunked prefill
+  - KV-blocked attention and ONNX subfunction export
+  - [GLM4-MOE Disaggregated Example](https://github.com/quic/efficient-transformers/blob/main/examples/disagg_serving/glm4_moe_disagg_mode_with_chunking.py)
+
+- **Qwen3-VL Embedding Model**
```
Comment (quic-rishinr) [inline docs/source/release_docs.md]: remove

File: docs/source/release_docs.md [hunk 9]
```diff
@@ -1,3 +1,233 @@
...
+  - Fixed `device_ids` type handling for single-device `qaicrt.Program`
+  - Clarified CausalLM decode-only flag handling and fixed PL=1 specialization naming
+  - Removed `custom_io` passing for BF16 dtype case
+
+---
+
+## Example Scripts
+
+- [Gemma 4 (Text + Multimodal)](https://github.com/quic/efficient-transformers/tree/main/examples/image_text_to_text/models/gemma_vision/gemma4)
+- [Kimi-K2 (MLA + MoE)](https://github.com/quic/efficient-transformers/tree/main/examples/kimi_k2)
```
Comment (quic-rishinr) [inline docs/source/release_docs.md]: remove

File: docs/source/validate.md [hunk 10]
```diff
@@ -39,7 +39,7 @@
 | **LlamaSwiftKVForCausalLM**  | swiftkv            | [Snowflake/Llama-3.1-SwiftKV-8B-Instruct](https://huggingface.co/Snowflake/Llama-3.1-SwiftKV-8B-Instruct)                                                  | ✔️          |
 | **Grok1ModelForCausalLM** ** |  grok-1②          | [hpcai-tech/grok-1](https://huggingface.co/hpcai-tech/grok-1)                                                  | ✕          |
 | **Glm4MoeForCausalLM**  |     GLM-4.5       | [zai-org/GLM-4.5](https://huggingface.co/zai-org/GLM-4.5)                                                  |          |
-
+| DeepseekV3ForCausalLM | Kimi k2| [moonshotai/Kimi-K2-Thinking](https://huggingface.co/moonshotai/Kimi-K2-Thinking)|
```
Comment (quic-rishinr) [inline docs/source/validate.md]: remove

File: README.md [hunk 11]
```diff
@@ -6,8 +6,22 @@
 ---
 
 *Latest news* :fire: <br>
+- [06/2026] Added support for Gemma4 models, [google/gemma-4-E2B-it](https://huggingface.co/google/gemma-4-E2B), [google/gemma-4-26B-A4B-it](https://huggingface.co/google/gemma-4-26B-A4B-it)
+- [06/2026] Added support for Qwen3.5 MoE model [Qwen/Qwen3.6-35B-A3B](https://huggingface.co/Qwen/Qwen3.6-35B-A3B)
+- [06/2026] Added support for Qwen3.5 model[Qwen/Qwen3.5-0.8B](https://huggingface.co/Qwen/Qwen3.5-0.8B)
+- [06/2026] Added aupport for Qwen3 VL and Qwen3 MoE VL models, [Qwen/Qwen3-VL-30B-A3B-Instruct](https://huggingface.co/Qwen/Qwen3-VL-30B-A3B-Instruct), [Qwen/Qwen3-VL-32B-Instruct](https://huggingface.co/Qwen/Qwen3-VL-32B-Instruct)
+- [06/2026] Added support for Qwen3 VL Reranker model, [Qwen/Qwen3-VL-Reranker-8B](https://huggingface.co/Qwen/Qwen3-VL-Reranker-8B)
+- [06/2026] Added support for Kimi-K2 model [moonshotai/Kimi-K2-Thinking](https://huggingface.co/moonshotai/Kimi-K2-Thinking)
```
Comment (quic-rishinr) [inline README.md]: remove

File: README.md (hunk 3 above)
Comment (quic-rishinr) [reply README.md]: Lets make the changes for other occurrence of AI100

**Review by quic-rishinr — CHANGES_REQUESTED** [review]

File: docs/source/introduction.md (hunk 4 above)
Comment (abukhoy) [reply docs/source/introduction.md:28]: it's already added in the name of Qen3.5 MOE models.

File: docs/source/introduction.md (hunk 6 above)
Comment (abukhoy) [reply docs/source/introduction.md]: It's not updated, maybe line change issue.

**Review by quic-hemagnih — APPROVED** [review]
Share the local hosted github page for reveiw

File: docs/source/introduction.md [hunk 12]
```diff
@@ -20,12 +20,21 @@ For other models, there is comprehensive documentation to inspire upon the chang
 7. Sample example applications and demo notebooks
 8. Unit test templates.
 
-***Latest news*** : <br>
 
-- [coming soon] Support for more popular [models](models_coming_soon)<br>
+*Latest news* :fire: <br>
+- [06/2026] Added support for Gemma4 models, [google/gemma-4-E2B-it](https://huggingface.co/google/gemma-4-E2B), [google/gemma-4-26B-A4B-it](https://huggingface.co/google/gemma-4-26B-A4B-it)
+- [06/2026] Added support for Qwen3.5 MoE model [Qwen/Qwen3.6-35B-A3B](https://huggingface.co/Qwen/Qwen3.6-35B-A3B)
```
Comment (quic-hemagnih) [inline docs/source/introduction.md]: Can you change it to QWEN3.6 here

File: docs/source/introduction.md (hunk 6 above)
Comment (quic-hemagnih) [reply docs/source/introduction.md]: From where this change is coming?

File: docs/source/introduction.md [hunk 13]
```diff
@@ -52,17 +56,20 @@ For other models, there is comprehensive documentation to inspire upon the chang
...
 - [11/2024] [finite adapters support](https://github.com/quic/efficient-transformers/pull/153) allows mixed adapter usage for peft models.
 - [11/2024] [Speculative decoding TLM](https://github.com/quic/efficient-transformers/pull/119) QEFFAutoModelForCausalLM model can be compiled for returning more than 1 logits during decode for TLM.
 - [11/2024] Added support for [Meta-Llama-3.3-70B-Instruct](https://huggingface.co/meta-llama/Llama-3.3-70B-Instruct), [Meta-Llama-3.2-1B](https://huggingface.co/meta-llama/Llama-3.2-1B) and [Meta-Llama-3.2-3B](https://huggingface.co/meta-llama/Llama-3.2-3B)
-- [09/2024] [AWQ](https://arxiv.org/abs/2306.00978)/[GPTQ](https://arxiv.org/abs/2210.17323) 4-bit quantized models are supported
+- [09/2024] [AWQ](https://arxiv.org/abs/2306.00978)/[GPTQ](https://arxiv.org/abs/2210.17323) 4-bit quantized models are supported <br>
 - [09/2024] Now we support [PEFT](https://huggingface.co/docs/peft/index) models
-- [09/2024] Added support for [Gemma-2-Family](https://huggingface.co/collections/google/gemma-2-release-667d6600fd5220e7b967f315)
+- [01/2025] Added support for [Ibm-Granite] (https://huggingface.co/ibm-granite/granite-3.1-8b-instruct)
```
Comment (quic-hemagnih) [inline docs/source/introduction.md]: Repeated

**Review by quic-hemagnih — CHANGES_REQUESTED** [review]
Please check comments

**Review by quic-hemagnih — APPROVED** [review]
